internal/sequencerapi/errors.go

```diff
@@ -0,0 +1,20 @@
+package sequencerapi
+
```

Comment:
We need this to get access to the error code? Its easier to introduce something rather than modify the `rpc.Error`?

Reply:
ah yea good point, making `rpc.JsonError` public would be sufficient and better. Also very useful when using the rpc library externally in other services as well